Tests/Pages/User/ViewTest.php

```diff
@@ -18,12 +18,12 @@ function testWebmentionContent()
                 $notification = $eventdata['notification'];
             });
 
-            $source = 'http://karenpage.com/looking-for-information-' . md5(time() . rand(0, 9999));
+            $source = 'http://karenpage.dummy/looking-for-information-' . md5(time() . rand(0, 9999));
```

Comment:
Consider using a dataProvider

Reply:
We're not actually iterating here, so I'm not sure there's call for one?

Reply:
Yeah, my bad. Fighting through a cold!